tests/test_helpers.py

```diff
@@ -155,5 +164,42 @@ def test_subsol():
     assert_allclose(helpers.subsol(dt.datetime(2100, 12, 31, 23, 59, 59)),
                     (-23.021061422069053, -179.23129780639425))
 
+
+def datetime64_to_datetime(dt64):
+    """Convert numpy datetime64 object to a datetime datetime object.
+
+    Notes
+    -----
+    Works outside 32 bit int second range of 1970
+
+    """
+    year_floor = dt64.astype('datetime64[Y]')
+    month_floor = dt64.astype('datetime64[M]')
+    day_floor = dt64.astype('datetime64[D]')
+    year = year_floor.astype(int) + 1970
+    month = (month_floor - year_floor).astype('timedelta64[M]').astype(int) + 1
+    day = (day_floor - month_floor).astype('timedelta64[D]').astype(int) + 1
+    return dt.datetime(year, month, day)
+
+
+def test_subsol_array():
+    """Verify subsolar point calculation using an array of np.datetime64.
+
+    Notes
+    -----
+    Tested by ensurnig the array of np.datetime64 is equivalent to converting
```

Comment:
spelling error: tested by *ensuring*

Reply:
😛  Thanks for the catch!  In the future, for something like this you can use the suggestion button (the +/- button that appears before the different font styles in the comment header) to make a suggestion that the PR author can commit directly.  Lot of people also like it because GitHub then gives you credit for the commit.

Reply:
Okay thanks for the tip